src/run.rs

```diff
@@ -206,6 +212,24 @@ impl Uiua {
     pub fn mode(&self) -> RunMode {
         self.mode
     }
+    /// Set the command line arguments
+    pub fn with_args(mut self, args: Vec<String>) -> Self {
+        self.cli_arguments = args;
+        self
+    }
+    /// Get the command line arguments
+    pub fn args(&self) -> Vec<String> {
+        self.cli_arguments.clone()
+    }
+    /// Set the path of the file that is being executed
+    pub fn with_file_path(mut self, file_path: String) -> Self {
```

Comment:
Because `file_path` should be a `PathBuf`, this should take `impl Into<PathBuf>`.

Reply:
Ah nice, that's much more flexible.